script.js

```diff
@@ -0,0 +1,76 @@
+var drawing;
+var content;
+var winCombos;
+var turn = 0;
+var theCanvas;
+var c;
+var cxt;
+var squaresFilled = 0;
+var y;
+
+window.onload=function(){
+  drawing = new Array();
+  content = new Array();
+  winCombos = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]];
+  for(var l = 0; l <= 8; l=l+1){
+  drawing[l] = false;
+  content[l]='';
+  }
+};
+
+function canvasClicked(squareNumber){
+  theCanvas = "square"+squareNumber;
+  c = document.getElementById(theCanvas);
+  cxt = c.getContext("2d");
+  if(drawing[squareNumber-1] ===false){
+    if(turn%2===0){
+      cxt.beginPath();
+      cxt.moveTo(10,10);
+      cxt.lineTo(90,90);
+      cxt.moveTo(90,10);
+      cxt.lineTo(10,90);
+      cxt.stroke();
+      cxt.closePath();
```

Comment:
Wow. Nice drawing. I was just expecting the string "X" here (using css to make it big).  Feel free to take the easy way out for something like this (unless we specify a particular method).